contributors/devel/godep.md

```diff
@@ -90,6 +90,7 @@ rm -rf vendor
 ./hack/update-bazel.sh
 ./hack/update-godep-licenses.sh
 ./hack/update-staging-client-go.sh
+./hack/update-staging-godeps.sh
```

Comment:
Many people make assumptions about what should work, or what works for them in other environments. If we have explicit requirements, it's never a bad idea to call them out in comments if they are a common tripping point.

+1 to adding a comment.